Dockerfile.apache

```diff
@@ -8,12 +8,20 @@
 # Contributors:
```

Comment:
Note that if you edit your Dockerfiles in VSCode or Che, calling them "apache.Dockerfile" will allow them to get syntax highlighting automatically, but calling them Dockerfile.apache means you'll have to explicitly open them with the Dockerfile editor/plugin.

(That's why we have rhel.Dockerfile instead of Dockerfile.rhel in the other repos.) :D